main.C

```diff
@@ -887,26 +885,6 @@ void HandleTagImport (fstream &input, TAG *tag) {
   WriteApeTag(input, offsettag);
 }
 
-void HandleTagExport (TAG *tag) {
```

Comment:
I have no problem with removing this but it would be nice to mention the motivation in the commit message